### PR TITLE
Fix that the main facing of basic machines can be set to top/bottom

### PR DESCRIPTION
**Pull request by Dream-Master**

### CODE REVIEW

File: src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_BasicMachine.java [hunk 1]
```diff
@@ -109,11 +109,16 @@ public GT_MetaTileEntity_BasicMachine(String aName, int aTier, int aAmperage, St
         mGUIName = aGUIName;
         mNEIName = aNEIName;
     }
+
+    protected boolean isValidMainFacing(byte aSide) {
+    	return aSide > 1;
```
Comment (Technus) [inline src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_BasicMachine.java]: it looks like it does disallow top/bottom facing? Is that intended?

Comment (Dream-Master) [reply src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_BasicMachine.java]: @Antifluxfield

Comment (Dream-Master) [reply src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_BasicMachine.java]: I ask him and wait for an answer

Comment (Antifluxfield) [reply src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_BasicMachine.java]: It just disallow the “main facing”, or intuitively the "front side", the facing that you cannot connect wires and pipes on.

**Pull request merged**